zh/TOC.md

```diff
@@ -105,6 +105,7 @@
 + [TiDB Operator Roadmap](roadmap.md)
 + 版本发布历史
   + v1.2
+    - [1.2.0](releases/release-1.2.0.md)
```

Comment:
Should be consistent with the English version?

Reply:
Done